cds/modules/oauthclient/cern_openid.py

```diff
@@ -85,8 +85,8 @@ def find_remote_by_client_id(client_id):
 
 def fetch_extra_data(resource):
     """Return a dict with extra data retrieved from CERN OAuth."""
-    person_id = resource.get("cern_person_id")
-    return dict(person_id=person_id, groups=resource["groups"])
+    person_id = resource.get("cern_person_id") or resource.get("sub")
```

Comment:
Comment: Maybe we want to add a comment here stating that cern users will have `person_id` while non cern users will rely on the `sub`. Also I don't know what `sub` is, just wondering if there would be a better id to rely on?